src/goto-instrument/README.md

```diff
@@ -26,3 +27,157 @@ developers are advised to copy the directory, remove all files apart
+	        fptbl1[(signed long int)id1][(signed long int)id2];
+	        // 2 file function_pointer.c line 21 function func
+	        IF fptbl1[(signed long int)id1][(signed long int)id2] == f3 THEN GOTO 1
+	        // 3 file function_pointer.c line 21 function func
+	        IF fptbl1[(signed long int)id1][(signed long int)id2] == f2 THEN GOTO 2
+	        // 4 file function_pointer.c line 21 function func
+	        IF fptbl1[(signed long int)id1][(signed long int)id2] == f1 THEN GOTO 3
+	        // 5 file function_pointer.c line 21 function func
+	        IF fptbl1[(signed long int)id1][(signed long int)id2] == f4 THEN GOTO 4
+	        // 6 file function_pointer.c line 21 function func
+	        IF fptbl1[(signed long int)id1][(signed long int)id2] == g1 THEN GOTO 5
+	        // 7 file function_pointer.c line 21 function func
+	        IF fptbl1[(signed long int)id1][(signed long int)id2] == g2 THEN GOTO 6
+	        // 8 file function_pointer.c line 21 function func
+	     1: t=f3();
+	        // 9 file function_pointer.c line 21 function func
+	        GOTO 7
+	        // 10 file function_pointer.c line 21 function func
+	     2: t=f2();
+	        // 11 file function_pointer.c line 21 function func
+	        GOTO 7
+	        // 12 file function_pointer.c line 21 function func
+	     3: t=f1();
+	        // 13 file function_pointer.c line 21 function func
+	        GOTO 7
+	        // 14 file function_pointer.c line 21 function func
+	     4: t=f4();
+	        // 15 file function_pointer.c line 21 function func
+	        GOTO 7
+	        // 16 file function_pointer.c line 21 function func
+	     5: t=g1();
+	        // 17 file function_pointer.c line 21 function func
+	        GOTO 7
+	        // 18 file function_pointer.c line 21 function func
+	     6: t=g2();
+	        // 19 file function_pointer.c line 22 function func
+	     7: fptbl2[(signed long int)id1];
+	        // 20 file function_pointer.c line 22 function func
+	        IF fptbl2[(signed long int)id1] == g1 THEN GOTO 8
+	        // 21 file function_pointer.c line 22 function func
+	        IF fptbl2[(signed long int)id1] == g2 THEN GOTO 9
+	        // 22 file function_pointer.c line 22 function func
+	     8: t=g1();
+	        // 23 file function_pointer.c line 22 function func
+	        GOTO 10
+	        // 24 file function_pointer.c line 22 function func
+	     9: t=g2();
+	        // 25 file function_pointer.c line 23 function func
+	    10: return t;
+	        // 26 file function_pointer.c line 23 function func
+	        dead t;
+	        // 27 file function_pointer.c line 24 function func
+	        END_FUNCTION
+
+You can now see that the function pointer (indirect) call (resulting from 
+the array indexing of the array containing the function pointers) 
+has now been substituted by a dispatch table.
+
+### Call Graph ###
+
+This is an example of a command line flag that doesn't require two file
+arguments, one for input and one for output.
```

Comment:
How about: "This is an example of a command-line flag that requires only one argument, specifying the input file." (Talk about what is the case, not what is not.)

Reply:
Yes this is a good suggestion, I'll get down to it now.